.github/workflows/realm-empty-env.yml

```diff
@@ -0,0 +1,26 @@
+name: Realm Deploy to Empty Environment
+
+on: [workflow_dispatch]
+
+#on:
```

Comment:
commented code able to be removed?
I understand we want to keep this on manual workflow_dispatch and not run automatically on branch pushes.